### PR TITLE
BWDB-15635: Iris V2 API migration guide framework.

### PR DESCRIPTION
**Pull request by severhardt-bw**
**⚠️Please do not make any changes to spec files in this repository!⚠️**

**All spec changes should be done in the [api-specs](https://github.com/Bandwidth/api-specs) repository.**

**Please follow the [Guide](https://bandwidth-jira.atlassian.net/wiki/spaces/DX/pages/4098359409/How+to+Update+API+Specifications+and+Contribute+to+Developer+Documentation) for contributing to our developer documentation.**

### CODE REVIEW

**Review by ajrice6713 — CHANGES_REQUESTED** [review]

**Review by ajrice6713 — DISMISSED** [review]

File: site/docs/numbers/v2Migration.mdx [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+
+### Introduction
+V2 of the Numbers API introduces several changes that make it possible for Numbers to handle international numbers with country codes. This is done via the introduction of e164 format as the standard across the API.
+This format takes the form of a plus sign followed by a country code, followed by a full telephone number.
+ex. +1(980)5556666
+In the above example, +1 is the additional portion that represents this as a number belonging to the United States.
+In order to integrate with V2 of the Numbers API, you will need to make several changes to match this new format.
+- Update endpoint calls to use ‘v2’ instead of ‘v1’ in the routes
+- Update your callback subscriptions to be e164 compliant
+- Update your Jasper Report requests to use the v2 routes
```
Comment (mrmeyers99) [inline site/docs/numbers/v2Migration.mdx:27]: I think this is redundant.

File: site/docs/numbers/v2Migration.mdx [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+## Numbers v2 API
+
+### Introduction
+V2 of the Numbers API introduces several changes that make it possible for Numbers to handle international numbers with country codes. This is done via the introduction of e164 format as the standard across the API.
+This format takes the form of a plus sign followed by a country code, followed by a full telephone number.
+ex. +1(980)5556666
```
Comment (mrmeyers99) [inline site/docs/numbers/v2Migration.mdx:22]:
```suggestion
ex. +19805556666
```

File: site/sidebar.js [hunk 3]
```diff
@@ -65,6 +65,7 @@ module.exports = {
             },
           ],
         },
+        // "numbers/numbersApiV2",
```
Comment (mrmeyers99) [inline site/sidebar.js:68]: what is this for?

Comment (mrmeyers99) [reply site/sidebar.js:68]: we spoke in person and this will be uncommented when we're ready for customers to see it

**Review by mrmeyers99 — APPROVED** [review]

**Review by ajrice6713 — APPROVED** [review]

**Pull request merged**